garage/envs/mujoco/maze/maze_env.py

```diff
@@ -229,7 +227,7 @@ def get_ori(self):
             obj = obj.env
         try:
             return obj.get_ori()
-        except (NotImplementedError, AttributeError) as e:
+        except (NotImplementedError, AttributeError):
             pass
```

Comment:
I don't think it's ideal to pass an exception, maybe add a description

Reply:
I didn't write this code. Do you want me to still edit it?

Reply:
you can ignore it